README.md

```diff
@@ -16,72 +16,72 @@ Backed by Y Combinator
 
-Client libraries speed up internal development, and help acquire customers who use your API. Our auto-generated SDKs are idiomatic and feel handwritten.
+In short, Fern takes care of the boilerplate associated with developing APIs, so you can **focus on delivering business value**.
 
-### 🌿 Server-side code generation
+# Capabilities
 
-We automatically generate lots of boilerplate on the server side, like Pydantic models for FastAPI and Jersey interfaces for Spring Boot. We also add compile-time validation that all your endpoints are being served correctly.
+Currently, the Fern compiler can generate the following types of artifacts:
 
-### 🌿 Postman Collection
+### 🌿 Client Libraries and SDKs
 
-Complete with examples of successful and unsuccessful requests!
+Fern generates idiomatic SDKs you can use to interact with your APIs, or publish for third party usage. This makes keeping your client libraries up to date with your API endpoints trivial, and speeds up your workflows dramatically.
 
-### 🌿 An OpenAPI spec
+### 🌿 Backend Code Generation
 
-You can feed the generated OpenAPI into the endless list of tools that support OpenAPI.
+Define your API, and Fern will generate the boilerplate for you. All the generated code is validated at build time, ensuring your all your endpoints and their data are being served correctly.
 
-# Comparison with OpenAPI
+### 🌿 Developer Resources
 
-OpenAPI is a great tool for documenting APIs, but falls short for code generation use cases.
+Fern supports the generation of development resources such as Postman collections, and OpenAPI specifications, making integration into existing development workflows effortless.
 
-Read [Fern vs. OpenAPI](https://buildwithfern.com/docs/comparison-with-openapi)
-for an in-depth comparison. **TL;DR: we differ from OpenAPI in these areas:**
+## Fern and OpenAPI
 
-- [Simplicity](https://www.buildwithfern.com/docs/comparison-with-openapi#simplicity)
-- [New features in specification](https://www.buildwithfern.com/docs/comparison-with-openapi#new-features-in-specification)
-- [Quality of code generation](https://www.buildwithfern.com/docs/comparison-with-openapi#quality-of-code-generation)
-- [Focus on server-side API development](https://www.buildwithfern.com/docs/comparison-with-openapi#focus-on-server-side-api-development)
-- [Change management](https://www.buildwithfern.com/docs/comparison-with-openapi#change-management)
-- [Cloud-based code generation and publishing](https://www.buildwithfern.com/docs/comparison-with-openapi#cloud-based-code-generation-and-publishing)
+We have designed Fern to complement existing OpenAPI toolchains and workflows, rather than replace them. We believe OpenAPI does a good job at as a declarative standard for defining APIs, but at times it is too verbose and complex, which harms the quality of automatically generated code.
```

Comment:
We have designed Fern to complement existing OpenAPI tools and workflows rather than replace them. We believe that OpenAPI has helped the development community by providing a standard for expressing REST APIs. Still, at times it is verbose and has [footguns](https://en.wiktionary.org/wiki/footgun) that, if used, make it impossible to generate idiomatic code.

Reply:
> make it impossible to generate idiomatic code 

This is probably too exaggerated